### ISSUE TITLE
Poor web3.js query performace

### ISSUE THREAD

**Issue by wellcomez** [paritytech/parity#8594]
while i use web3.js or curl directly , I find its performance is quite poor for example, I can just finish 9 or 8 times call for getblockbynumber  in 1 second. And any other api has similar problem. Is there some way can improve it ?

But if i use command line to export ,its speed raises up more. 

_Before filing a new issue, please **provide the following information**._

➜  parity git:(master) ✗  target/release/parity  --version
Parity
  version Parity/v1.11.0-unstable-b0cc44a-20180421/x86_64-linux-gnu/rustc1.25.0
Copyright 2015, 2016, 2017, 2018 Parity Technologies (UK) Ltd
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>.
on Ubuntu 16.04 ,

**Comment by Tbaut** [comment]
Please be more specific/provide a call example. What performance would you expect?

**Comment by wellcomez** [comment]
I tried two ways to export 
one: by construct http request like method eth_getBlockByNumber , as a result 5 block will received every 1 second, I also try web3.js , result is similar.
tow: use command export xxxx ,additionary  I change its code and make export data in json format , as a result it can export 200~300 blocks every 1 second..
From results, I found web rpc's performance is too poor. 

I also try multi clients to send request to parity, but it just increases parity's cpu usage and there is no any performance improvement.
